framework/src/media/Kconfig

```diff
@@ -8,7 +8,6 @@ config MEDIA
 	default n
 	depends on AUDIO
 	select HAVE_CXX
-	select HAVE_CXXINITIALIZE
```

Comment:
Could you leave where this is concerned in a Kconfig in the commit description?

Reply:
Sorry, I didnt understand. Do I need to update the commit description on why I removed this line?

Reply:
Because this `select` is done in other config, we remove this.
I want to leave
1. why this config should be enabled
2. who enables this automatically instead of here

Reply:
I have updated the commit description.